### PR TITLE
Update test data

### PR DESCRIPTION
**Pull request by dr0i**
Some more 035 now gets the (correct) DE-605 prefix.
Note that e.g. HT016709661 got new POR(tfolio) entries.

Note that HT016709661.json got multiple redundant "modifiedBy" entries.
Will create a new issue for the latter.

### CODE REVIEW

**Comment by dr0i** [comment]
Assigning @TobiasNx for being aware of the changes. Awaiting +1 to merge it into master.

File: src/test/resources/alma/HT016709661.json [hunk 1]
```diff
@@ -12,6 +12,16 @@
       "label" : "lobid Organisation"
     },
     "id" : "https://lobid.org/item/990184127410206441"
+  }, {
+    "id" : "https://lobid.org/item/53482457280006441",
+    "type" : [ "Item" ],
+    "label" : "Portfolio",
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-605#!",
+      "label" : "lobid Organisation"
```
Comment (dr0i) [inline src/test/resources/alma/HT016709661.json:21]: Here it's always DE-605, while at least one entry in the XML indicates "Düsseldorf" and "Dortmund(?)" while having an ID for DE-605.

Comment (dr0i) [reply src/test/resources/alma/HT016709661.json:21]: (mention this in https://github.com/hbz/lobid-resources/issues/1135)

File: src/test/resources/alma/HT016709661.xml [hunk 2]
```diff
@@ -164,15 +164,39 @@
...
     <subfield code="b">2020-11-23 03:27:51 Europe/Berlin</subfield>
   </datafield>
+  <datafield tag="POR" ind1=" " ind2=" ">
+    <subfield code="x">System</subfield>
+    <subfield code="a">53482457280006441</subfield>
+    <subfield code="M">49HBZ_NETWORK</subfield>
+    <subfield code="p">61480999250006441</subfield>
+    <subfield code="q">Lizenzfreie E-Ressourcen der ZDB, die nicht &#252;ber die EZB verwaltet werden (lzzdb ohne URL der UR)</subfield>
+    <subfield code="c">static</subfield>
+    <subfield code="A">49HBZ_DUE</subfield>
```
Comment (dr0i) [inline src/test/resources/alma/HT016709661.xml:186]: Here you see indicating Düsseldrof by "<subfield code="A">49HBZ_DUE</subfield>"  while mapped to DE-605 by taking  <subfield code="a">53482457280006441</subfield> (see 5 lines up) into account.
What does the POR__.A means anyway?

Comment (TobiasNx) [reply src/test/resources/alma/HT016709661.xml:186]: https://github.com/hbz/lobid-resources/issues/1135#issuecomment-787710851

While "M" is the Member Code, "A" is a refrence for "Available for Institution". I don't know if this relevant for us.

**Comment by TobiasNx** [comment]
+1

**Comment by dr0i** [comment]
Merged, closed.

**Pull request merged**